warp/Network/Wai/Handler/Warp/Request.hs

```diff
@@ -209,7 +209,8 @@ type BSEndo = ByteString -> ByteString
 type BSEndoList = [ByteString] -> [ByteString]
 
 data THStatus = THStatus
-    {-# UNPACK #-} !Int -- running total byte count
+    {-# UNPACK #-} !Int -- running total byte count (excluding current header chunk)
```

Comment:
I believe that `UNPACK` here is not necessary because of `unbox-small-strict-fields`. Please check it out. If not necessary, please remove them.

Reply:
I just kept it like it was. I don't really know when/if `unbox-small-strict-fields` triggers, but I don't think keeping the pragma there would hurt, right?
I'm fine with whatever's decided, though, I don't have much of an opinion on this.

Reply:
@snoyberg Do you have an opinion about this?

Reply:
I have a small preference towards keeping the pragma for future proofing, but I’d defer to @kazu-yamamoto on this.

Reply:
I would like to remove `UNPACK`.

This is not relating to this topic. Since we enabled `Strict` and `StrictData` by default, I would like to remove all `!` someday.

Reply:
Ok, I removed the UNPACK pragmas.